Add new gall guide. (#1249)
urbit/urbit.org: content/docs/userspace/gall-guide/12-next-steps.md

+++
title = "12. Next Steps"
weight = 60
template = "doc.html"
+++

We've now covered all the arms of a Gall agent, and everything you need to know
to start writing your own agent.

> The one thing we haven't touched on is front-end development and integration,

Reviewer: ```suggestion
The things haven't touched on yet are front-end development and integration,
```
